libs/testing/cypress/browser/src/lib/browserAuth.ts

```diff
@@ -19,4 +19,12 @@ export const browserAuth = {
       indexedDB.deleteDatabase('firebaseLocalStorageDb');
     });
   },
+
+  getAuth() {
+    cy.window().should('have.property', 'LoginService');
+    return cy.window().then(w => {
```

Comment:
```suggestion
    return cy.window().then(w => w['LoginService'] as AuthService)
```
But I don't like the idea of user the service for that. I would prefer having something that is not related to Angular.